### PR TITLE
remove last insert row id

### PR DESCRIPTION
**Pull request by Raafas**
# Description

Remove `lastInsertedId` and create a trigger for insert.

### CODE REVIEW

File: Pod/SQL/SQLTableSource.m [hunk 1]
```diff
@@ -234,15 +238,40 @@ - (CIRStatement *)deleteStatement
 
 #pragma mark Execute
 
-- (BOOL)insert:(NSDictionary <NSString *, id> *)values error:(NSError **)error
+- (int)insert:(NSDictionary <NSString *, id> *)values error:(NSError **)error
 {
+	CIRDatabase *db = _databaseProvider.database;
```
Comment (giovanepossebon) [inline Pod/SQL/SQLTableSource.m]: Where do you need that?

File: Pod/SQL/SQLTableSource.m [hunk 2]
```diff
@@ -234,15 +238,40 @@ - (CIRStatement *)deleteStatement
...
+	NSNumber *i;
```
Comment (rafael-silva) [inline Pod/SQL/SQLTableSource.m:243]: I? Can you describe better this atribute?

File: Pod/SQL/SQLTableSource.m [hunk 3]
```diff
@@ -234,15 +238,40 @@ - (CIRStatement *)deleteStatement
...
+
+	sqlite3 *handler = _databaseProvider.database.handler;
+
+	int code = 0;
+	const char *errMessage;
+
+	code = sqlite3_exec(handler, [@"DROP TABLE IF EXISTS temp._temp" UTF8String], 0, 0, &errMessage);
```
Comment (rafael-silva) [inline Pod/SQL/SQLTableSource.m:248]: About this trigger I think there is no place for building, just execute, can you build a method for create triggers same insert, update, delete etc...

**Review by brunodlz — APPROVED** [review]

File: Pod/SQL/SQLTableSource.m (hunk 1 above)
Comment (Raafas) [reply Pod/SQL/SQLTableSource.m]: I don't

**Review by giovanepossebon — APPROVED** [review]

**Pull request merged**